### DISCUSSION TITLE
Are custom directives possible?

### DISCUSSION THREAD

**Discussion by ajgagnon** [WordPress/gutenberg#57281] [Interactivity API]
In the interactivity API, is it possible to add custom directives? I noticed `data-wp-each` is not in v1 (not sure if it's still planned), but it would be beneficial to understand if this is possible to add custom functionality.

Thank you!

**Comment by luisherranz** [comment]
> is it possible to add custom directives? 

Yes, absolutely. But not in the initial version.

> I noticed `data-wp-each` is not in v1

We finally decided to add it, and it's already [on the 1.0 roadmap](https://github.com/WordPress/gutenberg/issues/56803). 

We are already [testing it in Woo](https://github.com/woocommerce/woocommerce/pull/42930), and we'll add it in January to Gutenberg 🙂

**Reply by ajgagnon** [reply]
Nice! Awesome news!
